### ISSUE TITLE
eyes-storybook should fail the build on diffs

### ISSUE THREAD

**Issue by shlomitc** [applitools/eyes-storybook#9]
I'm using the `eyes-storybook` in my build process, and I would like it to close with an error in case of eyes diff.
Currently, it shows an error message, but keeps running for the next command.
For example in my `package.json`:
`test:eyes-storybook": "eyes-storybook -c ./.storybook-test && 'echo should not echo'",`
```
⇒  npm run test:eyes-storybook

> wix-style-react@6.12.3 test:eyes-storybook /Users/shlomitc/Projects/wix-style-react
> eyes-storybook -c ./.storybook-test && echo 'should not echo'

Using @applitools/eyes-storybook version 2.5.2.

✔ Storybook was started
✔ Reading stories
✖ Done 42 stories out of 42

[EYES: TEST RESULTS]:
Accordion: simple [1024x768] - Passed
AddItem/disabled theme: image [1024x768] - Passed
MessageBox/Alert: FootNote [1024x768] - Passed
Accordion: withButton [1024x768] - Passed
AddItem/size: tiny [1024x768] - Passed
MessageBox/Alert: Actions [1024x768] - Passed
AddItem/size: small [1024x768] - Passed
MessageBox/AlertRTL: Actions [1024x768] - Passed
Accordion: multiple [1024x768] - Failed 1 of 1
AddItem/size: medium [1024x768] - Passed
MessageBox/Destructive: Standard [1024x768] - Passed
Accordion: inCard [1024x768] - Passed
AddItem/size: large [1024x768] - Passed
MessageBox/Destructive: Secondary [1024x768] - Passed
AddItem/alignItems: center [1024x768] - Passed
GooglePreview/basic: default render [1024x768] - Passed
MessageBox/Announcement: Standard [1024x768] - Passed
AddItem/alignItems: right [1024x768] - Passed
GooglePreview/description: no description [1024x768] - Passed
AddItem/alignItems: left [1024x768] - Passed
MessageBox/Announcement: Footnote [1024x768] - Passed
GooglePreview/description: multiline [1024x768] - Passed
AddItem/theme: dashes [1024x768] - Passed
MessageBox/Announcement: DisabledAction [1024x768] - Passed
MessageBox/Alert: EmptyState [1024x768] - Passed
AddItem/theme: plain [1024x768] - Passed
MessageBox/Announcement: PremiumAction [1024x768] - Passed
MessageBox/Alert: Standard [1024x768] - Passed
MessageBox/AnnouncementRTL: Standard [1024x768] - Passed
AddItem/theme: filled [1024x768] - Passed
MessageBox/Alert: Secondary [1024x768] - Passed
MessageBox/AnnouncementRTL: Footnote [1024x768] - Passed
AddItem/theme: image [1024x768] - Passed
MessageBox/Alert: ImageWithSideAction [1024x768] - Passed
AddItem/disabled theme: dashes [1024x768] - Passed
MessageBox/AnnouncementRTL: DisabledAction [1024x768] - Passed
MessageBox/Alert: Scrollable [1024x768] - Passed
AddItem/disabled theme: plain [1024x768] - Passed
MessageBox/AnnouncementRTL: PremiumAction [1024x768] - Passed
MessageBox/Alert: Image [1024x768] - Passed
AddItem/disabled theme: filled [1024x768] - Passed
SocialPreview/basic: default render [1024x768] - Passed

A total of 1 difference was found.

See details at ...
Total time: 34 seconds

Important notice: the Applitools visual tests are currently running with a concurrency value of 10.
This means that only up to 10 visual tests can run in parallel, and therefore the execution might be slow. This is the default behavior for free accounts.
If your account does support a higher level of concurrency, it's possible to pass a different value by specifying `concurrency:X` in the applitools.config.js file.
For more information on how to configure the concurrency level, visit the following link: https://www.npmjs.com/package/@applitools/eyes-storybook#concurrency.
If you are interested in speeding up your visual tests, contact sdr@applitools.com to get a trial account and a higher level of concurrency.

should not echo
```

as you can see `should not echo` is printted although there was an error

**Comment by danielschwartz85** [comment]
Thanks for the issue - definitely should return a failure exit code in case of failure.
WIP.

**Comment by helguita** [comment]
We did a quick solve for this in the meantime by doing:

`npm run eyes-storybook | grep -q 'A total of' && exit 1`

**Comment by danielschwartz85** [comment]
Thanks for the issue !
@helguita  @shlomitc  - fixed now in ```@applitools/eyes-storybook@2.6.4```
Cheers.
